glados/tts.py

```diff
@@ -216,9 +218,11 @@ class Phonemizer:
     espeakVOICE = "en-us"
 
     def __init__(self):
-        self.libc = ctypes.cdll.LoadLibrary("libc.so.6")
+        # self.libc = ctypes.cdll.LoadLibrary("libc.so.6")
+        self.libc = ctypes.cdll.LoadLibrary("libc.dylib")
         self.libc.open_memstream.restype = ctypes.POINTER(ctypes.c_char)
-        self.lib_espeak = self._load_library("libespeak-ng.so", "libespeak-ng.so.1")
+        #self.lib_espeak = self._load_library("libespeak-ng.so", "libespeak-ng.so.1")
+        self.lib_espeak = self._load_library("libespeak.dylib", "libespeak.1.dylib")
```

Comment:
What worked for me on MacOS was
```
import platform
[...]
self.lib_espeak = ctypes.cdll.LoadLibrary("libespeak-ng.{}".format("so.1" if platform.uname()[0] != "Darwin" else "dylib"))
```

Still need to test on linux, will do later today